packages/template/-private/dsl/types.d.ts

```diff
@@ -10,7 +10,7 @@ export type ResolveOrReturn<T> = T & (<U>(item: U) => () => U);
  */
 export type ElementForTagName<Name extends string> = Name extends keyof HTMLElementTagNameMap
   ? Name extends keyof SVGElementTagNameMap
-    ? HTMLElementTagNameMap[Name] & SVGElementTagNameMap[Name]
+    ? HTMLElementTagNameMap[Name] | SVGElementTagNameMap[Name]
```

Comment:
comment also says that it should be an union `This will return a union for elements that exist both in HTML and SVG.`
But it was an intersection